api/src/main/java/io/minio/MinioClient.java

```diff
@@ -4295,20 +4295,53 @@ public void putObject(
    * @throws NoSuchAlgorithmException thrown to indicate missing of MD5 or SHA-256 digest library.
    * @throws XmlParserException thrown to indicate XML parsing error.
    */
+  @Deprecated
```

Comment:
Good to add a javadoc comment with `@deprecated` also providing link to the alternate method to be used

Reply:
Done